packages/card/src/types.ts

```diff
@@ -0,0 +1,22 @@
+export type FzCardProps = {
+  title: string;
+  color?: FzCardColor;
+  primaryAction?: FzCardButton;
+  secondaryAction?: FzCardButton;
+  tertiaryAction?: FzCardIconButton;
```

Comment:
What's the benefit of taking objects as props? I think primitive ones are easier to pass

Reply:
I left the object in case you want to extend it with other properties in the future